openspec/changes/docs-13-core-nav-search-theme-roles/TDD_EVIDENCE.md

```
@@ -0,0 +1,105 @@
+PYTHONPATH=/home/dom/git/nold-ai/specfact-cli-worktrees/feature/docs-13-core-nav-search-theme-roles/src \
+/home/dom/git/nold-ai/specfact-cli/.venv/bin/python -m pytest \
+  tests/unit/test_core_docs_site_contract.py \
+  tests/unit/docs/test_release_docs_parity.py \
+  tests/unit/docs/test_docs_validation_scripts.py -q
+```
+
+Result:
+
+- `38 passed`
+
+2. Docs command validation
```

markdownlint-cli2 (0.22.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)